ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
+    "filename": "jquery.loadTemplate.min.js",
+    "keywords": [
+       "templates",
+       "templating",
+       "jquery-plugin"
+    ],
+    "author": "codepb",
+    "homepage": "http://codepb.github.io/jquery-template/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com:codepb/jquery-template.git"
```

Comment:
The `url` can't redirect to the library GitHub page.
Let's replace `:` to `/`, as the followings:
`git+https://github.com/codepb/jquery-template.git`
Thank you.